lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -368,12 +429,29 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
                   value={filterOption}
                 >
                   {this.messages(`filter.${filterOption}`)}{' '}
-                  <Badge
-                    style={{backgroundColor: '#babec0'}}>
+                  <Badge style={{ backgroundColor: '#babec0' }}>
                     {filterCounts[filterOption]}
                   </Badge>
                 </OptionButton>
               ))}
+              {project.labels.length > 0 && (
+                <DropdownButton
+                  id='project-feedsource-label-filter-button'
+                  // to match the buttons with badges
+                  style={{ paddingBottom: 7 }}
+                  title={
+                    <span
+                      style={{
+                        fontWeight: activeFilterLabelCount > 0 ? 'bold' : ''
```

Comment:
This bold text is very hard to distinguish for me. I don't have an alternative recommendation at the moment.

Reply:
Have replaced `bold` with `900`, which should help! Otherwise I was thinking either coloring it or putting a little dot next to labels or something. The only issue is that the bar is so long it is very close to breaking over to a new line, so horizontal space is at a premium